nautobot_circuit_maintenance/views.py

```diff
@@ -5,16 +5,28 @@
 
 import google_auth_oauthlib
 from django.conf import settings
+from django.contrib import messages
 from django.shortcuts import redirect
 from django.urls import reverse
 from django.urls.exceptions import NoReverseMatch
-from nautobot.circuits.models import Circuit, Provider
+from nautobot.apps.views import NautobotUIViewSet
+from nautobot.circuits.models import Circuit
 from nautobot.core.views import generic
+from nautobot.core.views.mixins import (
+    ObjectBulkDestroyViewMixin,
+    ObjectDestroyViewMixin,
+    ObjectDetailViewMixin,
+    ObjectListViewMixin,
+)
+from rest_framework.decorators import action
+from rest_framework.response import Response
 
 from nautobot_circuit_maintenance import filters, forms, models, tables
 from nautobot_circuit_maintenance.handle_notifications.sources import RedirectAuthorize, Source
 from nautobot_circuit_maintenance.models import CircuitMaintenance
 
+from .api import serializers
```

Comment:
Use the full reference. Usually we do

```
from nautobot_circuit_maintenance import filters, forms, models, tables
from nautobot_circuit_maintenance.api import serializers
```

can remove this as well: 
`from nautobot_circuit_maintenance.models import CircuitMaintenance` and replace with `models.CircuitMaintenance`